openpower/package/openpower-vpnor/openpower-vpnor.mk

```diff
@@ -0,0 +1,14 @@
+################################################################################
+#
+# openpower_vpnor
+#
+################################################################################
+
+HOST_OPENPOWER_VPNOR_VERSION ?= 5c90711a8aaa6b1d760388a5116e36831ce5e7ab
```

Comment:
It's just the git commit ID. I don't think there's any plan to tag releases on this repo.